release/release_cmds.sh

```diff
@@ -50,14 +51,14 @@ create_artifacts() {
   # Make the signature. This requires human input
   gpg -u $APACHE_USERNAME@apache.org --armor --output $PROJECT_NAME-$VERSION.tar.gz.asc --detach-sign $PROJECT_NAME-$VERSION.tar.gz
   # Make hashes
+  for algo in 256 512; do
+    shasum -a $algo $PROJECT_NAME-$VERSION.tar.gz > $PROJECT_NAME-$VERSION.tar.gz.sha$algo
```

Comment:
@hbdeshmukh PS, I added these changes in this PR as well. We now will use un-cracked versions of SHA 😄